mlflow/langchain/__init__.py

```diff
@@ -65,7 +65,13 @@
 _TOOLS_DATA_FILE_NAME = "tools.pkl"
 _TOOLS_DATA_KEY = "tools_data"
 _MODEL_TYPE_KEY = "model_type"
-
+_UNSUPPORTED_MODEL_ERROR_MESSAGE = "MLflow langchain flavor only supports logging " + \
+"subclasses of langchain.chains.base.Chain and langchain.agents.agent.AgentExecutor " + \
+"instances, found {instance_type}"
```

Comment:
```suggestion
_UNSUPPORTED_MODEL_ERROR_MESSAGE = (
    "MLflow langchain flavor only supports logging "
    "subclasses of langchain.chains.base.Chain and langchain.agents.agent.AgentExecutor "
    "instances, found {instance_type}"
```

We can remove `+ \`.